client/platform/web-girder/views/Admin/Addons.vue

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+import {
+  computed,
+  defineComponent, ref, Ref,
+} from '@vue/composition-api';
+import { getAddons, postAddons } from 'platform/web-girder/api/configuration.service';
+
+
+export default defineComponent({
+  name: 'AddOns',
+  setup() {
+    const table: Ref<[string, string, string, boolean][]> = ref([]);
+    const forceDownload = ref(false);
+    const selected: Ref<{Name: string; Description: string; Installed: boolean}[]> = ref([]);
+    const headers: Ref<{text: string; value: string}[]> = ref([
+      { text: 'Name', value: 'Name' },
+      { text: 'Description', value: 'Description' },
+      { text: 'Installed', value: 'Installed' },
+    ]);
+    // First we need to download the CSV from github
+    const getData = async () => {
+      table.value = (await getAddons()).data;
+    };
+    const data = computed(() => table.value.map(
+      (item) => ({ Name: item[0], Description: item[2], Installed: item[3] }),
+    ));
+    getData();
+
+    const downloadAddons = async () => {
+      // Lets create a list of URLS to download to add
+      const downloadArray = selected.value.map((item) => {
+        const found = table.value.find((tabItem) => tabItem[0] === item.Name);
+        if (found !== undefined) {
+          return found[1];
+        }
+        return null;
+      });
+      const list = downloadArray.filter((item) => item !== null) as string[];
+      if (list.length) {
+        postAddons(list, forceDownload.value);
+      }
+    };
+    return {
+      selected,
+      table,
+      headers,
+      data,
+      forceDownload,
+      downloadAddons,
+    };
+  },
+
+});
+</script>
+
+<template>
+  <v-container>
+    <v-card>
+      <v-card-title> AddOns </v-card-title>
+      <v-card-text>
+        <p>
+          Below is a list of the available addons for VIAME.
+          Their current checked state is not an indication if
+          they are installed or not, just if you want to download and install the addons.
+          If you don't use the "Force Download" switch it won't attempt
+          to download addons which are already installed
```

Comment:
I think the verbiage here needs to change a bit. Perhaps:
"Below is a list of the available configuration addons for working with VIAME. Select the addons you want to download and install. If an addon is already installed and you want to install again you must use the 'Force Download' switch"

Reply:
Additionally would we rather just apply the force download behind the scenes if the user has selected an addon which is already installed?

Reply:
I kind of want to keep the force download, They all appear to be Girder ItemIds which means the files could change inside of the item causing there to be an update but the filename wouldn't be different (the downloaded filenames are the girder ItemId).  So a force download may be needed in the future to actually upgrade the pipeline.  We can talk to Matt and try to ensure that all uploads are done by creating new itemIds instead of just replacing the files.